localization/gyro_odometer/README.md

```diff
@@ -21,10 +21,7 @@
 
 ## Parameters
 
-| Parameter             | Type   | Description                      |
-| --------------------- | ------ | -------------------------------- |
-| `output_frame`        | String | output's frame id                |
-| `message_timeout_sec` | Double | delay tolerance time for message |
+{{ json_to_markdown("localization/gyro_odometer/schema/gyro_odometer.schema.json") }}
```

Comment:
Hi @Motsu-san , I was following Step 6 in the [instruction](https://docs.google.com/document/d/10DQezFZ0XAjO2S0IG_6O36p1WhZI7be68QRhNirzdac/edit?pli=1#heading=h.z8xipsmunki6).

That document provided an example here ([link](https://autowarefoundation.github.io/autoware.universe/main/perception/lidar_apollo_segmentation_tvm_nodes/#parameters)) and I believe this is what it will look like.

Reply:
I get it. Thank you.
On github readme view, though that will be an unknown line, I confirmed you were following the instruction. 
![image](https://github.com/autowarefoundation/autoware.universe/assets/83898149/4f5e4f17-e0ff-4b12-9bb5-427c6162b555)